docs/components/processor.rst

```diff
@@ -47,6 +47,31 @@ crash id to the "socorro.normal" RabbitMQ queue.
 We have helper scripts for these steps.
 
 
+``scripts/fetch_crashids.py``
+-----------------------------
+
+This will generate a list of crash ids from -prod that meet specified criteria.
```

Comment:
The `-prod` nomenclature is a bit cryptic. I don't really know where it comes from but perhaps it'd be better to simply state `crash-stats.mozilla.com` instead.